frontend/src/utilities/notebookControllerUtils.ts

```diff
@@ -310,7 +311,8 @@ export const getNotebookEventStatus = (
 ): NotebookProgressStep => {
```

Comment:
Modify the `NotebookProgressStep` object -- we need to make `step` required.

In any place that used `description` instead of `step` (because it was optional) -- go ahead and use description instead -- because step will always be defined now.